web/translation/translate.fa_IR.toml

```diff
@@ -213,151 +213,151 @@
 "telegramNotifyTime" = "مدت زمان نوتیفیکیشن ربات تلگرام"
-"telegramNotifyTimeDesc" = "از فرمت زمان بندی لینوکس استفاده کنید "
-"tgNotifyBackup" = "پشتیبان گیری از پایگاه داده"
-"tgNotifyBackupDesc" = "ارسال کپی فایل پایگاه داده به همراه گزارش دوره ای"
+"telegramNotifyTimeDesc" = "از فرمت زمان‌بندی لینوکس استفاده کنید "
+"tgNotifyBackup" = "پشتیبان گیری از دیتابیس"
+"tgNotifyBackupDesc" = "ارسال کپی فایل دیتابیس به همراه گزارش دوره‌ای"
 "tgNotifyLogin" = "اعلان ورود"
-"tgNotifyLoginDesc" = "نام کاربری، آدرس ای پی، و زمان وقتی که فردی سعی می‌کند به پنل شما وارد شود نمایش میدهد"
+"tgNotifyLoginDesc" = "نام‌کاربری، آدرس آی‌پی، و زمان ورود، فردی که سعی می‌کند وارد پنل شود را نمایش می‌دهد"
 "sessionMaxAge" = "بیشینه زمان جلسه وب"
-"sessionMaxAgeDesc" = "بیشینه زمانی که میتوانید لاگین بمانید (واحد: دقیقه)"
+"sessionMaxAgeDesc" = "بیشینه زمانی که میتوانید لاگین بمانید. واحد: دقیقه"
 "expireTimeDiff" = "آستانه زمان باقی مانده"
-"expireTimeDiffDesc" = "فاصله زمانی هشدار تا رسیدن به زمان انقضا (واحد: روز)"
+"expireTimeDiffDesc" = "فاصله زمانی هشدار تا رسیدن به زمان انقضا. واحد: روز"
 "trafficDiff" = "آستانه ترافیک باقی مانده"
-"trafficDiffDesc" = "فاصله زمانی هشدار تا رسیدن به اتمام ترافیک (واحد: گیگابایت)"
-"tgNotifyCpu" = "آستانه هشدار درصد پردازنده"
-"tgNotifyCpuDesc" = "این ربات تلگرام در صورت استفاده پردازنده بیشتر از این درصد برای شما پیام ارسال می کند.(واحد: درصد)"
-"timeZone" = "منظقه زمانی"
-"timeZoneDesc" = "وظایف برنامه ریزی شده بر اساس این منطقه زمانی اجرا می شوند. پنل را مجدداً راه اندازی می کند تا اعمال شود"
+"trafficDiffDesc" = "فاصله زمانی هشدار تا رسیدن به اتمام ترافیک. واحد: گ.ب"
+"tgNotifyCpu" = "آستانه هشدار بار پردازنده"
+"tgNotifyCpuDesc" = "اگر استفاده از پردازنده از این آستانه فراتر رفت، برای شما پیام ارسال می‌شود. واحد: درصد"
+"timeZone" = "منطقه زمانی"
+"timeZoneDesc" = "وظایف برنامه ریزی شده بر اساس این منطقه زمانی اجرا می‌شود"
 "subSettings" = "سابسکریپشن"
-"subEnable" = "فعال کردن سرویس"
-"subEnableDesc" = "ویژگی سابسکریپشن با پیکربندی جداگانه"
-"subListen" = "محدودیت آی‌پی"
-"subListenDesc" = "برای استفاده از همه آی‌پی ها به طور پیش فرض خالی بگذارید"
-"subPort" = "پورت سرویس"
-"subPortDesc" = "شماره پورت برای ارائه خدمات سابسکریپشن"
-"subCertPath" = "مسیر فایل کلید عمومی گواهی سابسکریپشن"
-"subCertPathDesc" = "یک مسیر مطلق که با '/' شروع می شود را پر کنید."
-"subKeyPath" = "مسیر فایل کلید خصوصی گواهی سابسکریپشن"
-"subKeyPathDesc" = "یک مسیر مطلق که با '/' شروع می شود را پر کنید."
-"subPath" = "مسیر ریشه سابسکریپشن"
-"subPathDesc" = "باید با '/' شروع شود و با '/' ختم شود."
-"subDomain" = "دامنه مخصوص سابسکریپشن"
-"subDomainDesc" = "برای نظارت بر همه دامنه ها و آی‌پی ها به طور پیش فرض خالی بگذارید"
-"subUpdates" = "فاصله به روز رسانی های سابسکریپشن"
-"subUpdatesDesc" = "ساعت های فاصله بین به روز رسانی در برنامه کاربر"
-"subEncrypt" = "رمزگذاری کانفیگ ها"
-"subEncryptDesc" = "رمزگذاری کانفیگ های بازگشتی سابسکریپشن"
+"subEnable" = "سرویس سابسکریپشن"

```

Comment:
This is not a good translation. Enabling is not mentioned here

Reply:
It is true! But we should consider people who never focusing on details.
I had this experience several times even if I made it bold or with other colors.
In another hand, in settings the switch is not nearby the text.
Regarding text alignment and spaces, we have enough space for that.